app/views/products/show.html.erb

```diff
@@ -1,27 +1,43 @@
 <div class="row">
 	<div class="col-md-6">
-		<h2><%= @product.name %></h2>
 		<%= image_tag @product.image.url(:medium) %>
-		<p><strong>Category:</strong><br /><%= @product.category %></p>
-		<%= link_to 'Edit', edit_product_path(@product) %>
 	</div>
-	<div class="col-md-6 description">
-		<p><%= @product.description %></p>
+	<div class="col-md-6">
+		<div class="panel panel-default">
+  <div class="panel-heading">
```

Comment:
Hmm, it doesn't look that way in Sublime.
![screen shot 2016-02-16 at 1 42 36 pm](https://cloud.githubusercontent.com/assets/14825209/13090472/3df5c84e-d4b3-11e5-8b4c-2a4f8a695dd8.png)